libs/application/templates/health-insurance/src/healthInsuranceUtils.ts

```diff
@@ -8,26 +8,23 @@ export const hasHealthInsurance = (externalData: ExternalData) => {
   return isInsured === true
 }
 
-export const hasActiveApplication = (externalData: ExternalData) => {
+export const hasActiveDraftApplication = (externalData: ExternalData) => {
   const response = externalData?.applications
   if (response && response?.status === 'success') {
     const applications = response.data as Applications[]
     const pendingApplications = applications?.filter(
-      (application) =>
-        application.state === 'draft' ||
-        application.state === 'inReview' ||
-        application.state === 'missingInfo',
+      (application) => application.state === 'draft',
     )
     return pendingApplications?.length > 1
   }
   // If we can not find any pending applications becausee of failure to fetch info, we will return false to allow the user to continue to create a new application
   return false
 }
 
-export const hasOldPendingApplications = (externalData: ExternalData) => {
-  const oldPendingApplications = externalData?.oldPendingApplications
+export const pendingApplications = (externalData: ExternalData) => {
+  const pendingApplications = externalData?.oldPendingApplications
```

Comment:
the `id` of the field in HealthInsuranceForm should be changed as well so that it is not `externalData?.oldPending...`
I also think the actual provider file is namned old pending still